assets/src/components/routeLadders.tsx

```diff
@@ -41,14 +46,31 @@ const RouteLadders = ({
     }
   }
 
+  const routes: Route[] | null = useContext(RoutesContext)
+  const selectedRoutes: Route[] = selectedRouteIds
+    .map((routeId) => findRouteById(routes, routeId))
+    .filter((route) => route) as Route[]
+  const timepointsByRouteId: TimepointsByRouteId =
+    useTimepoints(selectedRouteIds)
+
+  const { socket } = useContext(SocketContext)
+  const alerts = useAlerts(socket, selectedRouteIds)
```

Comment:
Refactoring the alerts code here worries me a bit because it's completely untested, on `main` as well as here.

I tried clicking around to test this - I wasn't able to find any alerts visible in skate (either local or prod), so I didn't get a good manual check, but then just for fun, I went ahead and deleted it entirely, and once I removed all the unused imports and things, all of the tests still passed. I tried out the same thing on `main`, and same deal.

I would feel more comfortable with this if we snuck another PR in front of this one that added some tests that validated the alerts logic.

Reply:
Merged `main`, and the `useAlerts` test still passes ✅